src/App.js

```diff
@@ -19,13 +42,18 @@ const App = () => {
     }, [])
 
     return (
-    <div className="App">
-        <Header setSidebarIsHidden={setSidebarIsHidden} sidebarIsHidden={sidebarIsHidden} />
-        <main>
-            <SideBar sidebarIsHidden={sidebarIsHidden} allEmailSnippets={allEmailSnippets} />
-            <EmailCardList allEmailSnippets={allEmailSnippets} />
-        </main>
-    </div>
+        <div className="App">
+            <Header setSidebarIsHidden={setSidebarIsHidden} sidebarIsHidden={sidebarIsHidden}/>
+            <div className={'d-flex flex-row vh-100'}>
```

Comment:
the main tag here was changed to div - it is probably better to still keep it as main

Reply:
oops it's not the App one that is a main, it's the one under the Header